src/Controller/UserController.php

```diff
@@ -31,12 +32,11 @@ public function index(UserRepository $userRepository): Response
     /**
      * @Route("/new", name="new", methods={"GET","POST"})
      */
-    public function new(Request $request, UserPasswordEncoderInterface $passwordEncoder): Response
+    public function new(Request $request, UserPasswordEncoderInterface $passwordEncoder, EmailService $emailService): Response
```

Comment:
I think it would be better to inject the `EmailService` in the controller's constructor, and set it as a class attribute. Right now, we'll have to inject the service every time a new method needs to use the email service, but if it's a class attribute, we only inject it once and then use it in every method.

Reply:
Done